tests/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixerTest.php

```diff
@@ -642,6 +642,31 @@ public function testFnc()
         {
             aaa();
         }
+    }',
+            ],
+            'expecting exception with message below' => [
+                '<?php
+    class MyTest extends TestCase
+    {
+        /**
+         */
+        public function testSomething()
+        {
+            $this->setExpectedException(\Foo\Bar::class);
+
+        }
+    }',
+                '<?php
+    class MyTest extends TestCase
+    {
+        /**
+         * @expectedException Foo\Bar
+         *
+         * Testing stuff.
+         */
+        public function testSomething()
+        {
```

Comment:
let's add any line here to see that expectation was set before and not after it.

Reply:
friendly reminder, @kubawerlos